Dockerfile

```diff
@@ -38,7 +41,11 @@ RUN apt-get update -q && apt-get upgrade -q \
     libffi-dev \
     libssl-dev \
     vim-nox
```

Comment:
@moylop260 as I told you in the other pr I do a single clean up at [the end](https://github.com/vauxoo-dev/docker-ubuntu-base/blob/master-reduce_runs-truiz/Dockerfile#L49)